### ISSUE TITLE
CustomListItem: No aria-label support

### ISSUE THREAD

**Issue by sleric1024** [SAP/ui5-webcomponents#3014]
**Describe the bug**
When using the following component within List, we cannot using aria-label to define a message for a11y use case.
So there will be no indication that if there are interactive elements within each list item or not. 
- StandardListItem
- CustomListItem
- GroupHeaderListItem

**Expected behavior**
aria-label support for the following component
- StandardListItem
- CustomListItem
- GroupHeaderListItem

**Additional context**
I am from SAP SuccessFactors

**Comment by kskondov** [comment]
Hello @SAP/ui5-webcomponents-topic-p,  I am forwarding this issue to you, as owners of the ui5-list component

**Comment by ilhan007** [comment]
Hello @sleric1024 the solution will be part of 031.9 and 0.29.18. Check out the following PR for details https://github.com/SAP/ui5-webcomponents/pull/3269

**Comment by sleric1024** [comment]
Hi @ilhan007 , thank you for your kind reminding.
